protostar/cli/common_arguments.py

```diff
@@ -0,0 +1,8 @@
+from .protostar_argument import ProtostarArgument
+
+lib_path_arg = ProtostarArgument(
```

Comment:
Shouldn't this have a default of value "lib"?

Reply:
We can't use default if the ConfigurationFileV1 is used. Protostar needs to print the warning if the user decides to pass the argument explicitly and the Configuration File isn't migrated. LibPathResolver handles the default value logic for the time being.